lm_eval/api/task.py

```diff
@@ -1271,6 +1271,8 @@ def construct_requests(
         elif self.OUTPUT_TYPE == "multiple_choice":
             choices = self.doc_to_choice(doc)
             target_delimiter = self.config.target_delimiter
+            if kwargs.get("apply_chat_template", False) is True:
```

Comment:
This removes the target delimiter in multichoice tasks when using task templates. This is because the chat template acts as a target delimiter already.